src/components/PortfolioBalance/PortfolioBalance.js

```diff
@@ -21,37 +21,38 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import { View } from 'react-native';
 import { createStructuredSelector } from 'reselect';
-import type { Assets, Balances, Rates } from 'models/Asset';
+import type {
+  Balances,
+  Rates,
+} from 'models/Asset';
 import { BaseText } from 'components/Typography';
-import { calculatePortfolioBalance } from 'utils/assets';
+import { balanceInCurrency } from 'utils/assets';
 import { formatFiat } from 'utils/common';
 import { UIColors, baseColors, fontSizes } from 'utils/variables';
 import { defaultFiatCurrency } from 'constants/assetsConstants';
 import { accountBalancesSelector } from 'selectors/balances';
 
 type Props = {
-  assets: Assets,
   rates: Rates,
   balances: Balances,
   baseFiatCurrency: string,
   label?: string,
   style: Object,
 };
 
-class PortfolioBalance extends React.PureComponent<Props, {}> {
+class PortfolioBalance extends React.PureComponent<Props> {
   render() {
     const {
       style,
-      assets,
       rates,
       balances,
       baseFiatCurrency,
       label,
     } = this.props;
 
-    const portfolioBalances = calculatePortfolioBalance(assets, rates, balances);
     const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
-    const portfolioBalance = formatFiat(portfolioBalances[fiatCurrency], baseFiatCurrency);
+    const balance = balanceInCurrency(rates, balances, fiatCurrency);
```

Comment:
```suggestion
    const balance = balanceInFiat(rates, balances, fiatCurrency);
```

Reply:
When you read the code without knowing the details it could be confusing as we have another method with similar naming `formatFiat()`, that's why there was a prefix `calculate` as it shows the intention and you can assume what it does without going into the details

Reply:
What do you mean? What kind of confusion?

It's perfectly safe to assume that this function is not gonna calculate anything, just give you the balance in fiat; it won't make any difference.

Reply:
Updated.